src/main/webapp/creatives/maps/simid_map.html

```diff
@@ -1,13 +1,14 @@
 <html>
   <head>
-    <link rel="stylesheet" type="text/css" href="simid_maps.css">
+    <link rel="stylesheet" type="text/css" href="simid_map.css">
   </head>
   <body>
     <div id="container">
```

Comment:
maybe a more descriptive name like "button_container" rather than just container here

Reply:
@juliareichel do you mind if I change this?